pkgs/os-specific/linux/zfs/default.nix

```
@@ -8,8 +8,8 @@
 , libtirpc
 , nfs-utils
 , gawk, gnugrep, gnused, systemd
-, smartmontools, sysstat, sudo
-, pkg-config
+, smartmontools, enableMail ? false
+, sysstat, sudo, pkg-config
```

It is not clear for me which packages belong to enableMail.

Edit:

Maybe like this if we only need smartmontools.

```suggestion
, sysstat, sudo, pkg-config
, enableMail ? false, smartmontools
```